release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -147,6 +152,14 @@ The CNI VRF plug-in was previously introduced as a Technology Preview feature in
 
 For more information, see xref:../networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc#cnf-assigning-a-secondary-network-to-a-vrf[Assigning a secondary network to a VRF].
 
+[id="ocp-4-9-networking-openshift-on-openstack-cloud-provider-options"]
+==== Load balancer configuration as a cloud provider option for clusters on {rh-openstack}
+
+For clusters that run on {rh-openstack}, Octavia is now available as a default load balancer option. 
+
+// TODO: Link!
+// For more information, see xref:../
```

Comment:
I added a known issue about this feature into the tracker PR recently. :/ Here's the BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2004542.

Reply:
TY. :\